load-data-to-gsheet/README.md

```diff
@@ -0,0 +1,19 @@
+
+## Steps to run the sample
+
+### Obtaining Google sheets credentials
```

Comment:
Use the same convension as in other samples

Reply:
Updated please check.

Reply:
Updated in [139ba](https://github.com/ballerina-guides/etl-samples/pull/8/commits/139bab7a1b37c1f9ef4c163e1c7e23fd5b6e2ed5).